autoscale_cloudroast/test_repo/autoscale/system/schedule_policies/test_system_execute_scheduler_negative.py

```diff
@@ -152,7 +154,9 @@ def test_system_scheduler_batch(self):
             sp_list=at_style_policies_list)
         group = response.entity
         self.resources.add(group, self.empty_scaling_group)
-        sleep(self.scheduler_interval * 2)
+        # An extra 1 second to let scheduler processing take place due to
+        # excess events
+        sleep(10 + self.scheduler_interval + 1)
```

Comment:
Non-blocking question:  why the change from twice the scheduler interval to 10 + one interval + 1?  Is the scheduler interval just 10?